ibridges/__main__.py

```diff
@@ -318,3 +320,85 @@ def ibridges_sync():
                   _parse_str(args.destination, session),
                   dry_run=args.dry_run,
         )
+
+# prefix components:
+space =  '    '
+branch = '│   '
+skip = "..."
+# pointers:
+tee =    '├── '
+last =   '└── '
```

Comment:
consider/remember an ascii-only presentation.

Reply:
Yes, that sounds like a good idea. I have added an optional `--ascii` option for those that need it. Thanks!

Reply:
i see!  looks great.

Reply:
Thanks!